src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -30,7 +30,7 @@ class MarathonScheduler(
   private[this] val log = LoggerFactory.getLogger(getClass.getName)
 
   private var lastMesosMasterVersion: Option[SemanticVersion] = Option.empty
-  @volatile private[this] var homeRegion: Option[String] = Option.empty
+  @volatile private[this] var homeFaultDomain: Option[FaultDomain] = Option.empty
```

Comment:
Really do NOT like the name change to FaultDomain... this is not a fault domain.   there is no described fault domain... that is left to the user.   homeRegion makes sense... homeDomain... defaultDomain

Reply:
+1

Reply:
I would vote for keeping home region...

Reply:
We do call it `FaultDomain` though, don't we? @meichstedt what do the other teams call these? `Region` or `FaultDomain`?

Reply:
@kensipe LocalDomain?

Reply:
Here it is called home region https://docs.google.com/document/d/11qJtAR8aT7ZHrUjBwsDtet9FVeIGre7GQdppF9pbcdE/edit#heading=h.v9bqn0eo8mhe and I kind of like that :-)